### ISSUE TITLE
Update logo

### ISSUE THREAD

**Issue by MortenTobiasNielsen** [MortenTobiasNielsen/Factorio-GUI-V2#302]
We should make a new logo so it is FTG instead or maybe something actually creative. :D

**Comment by theis999** [comment]
I know that neither of us are graphical artist, so either: 
We ask for help
We find some helpful website such as [game-icons](https://game-icons.net/)
We do something very simple (similar to what we have)

**Comment by MortenTobiasNielsen** [comment]
Agreed. We need to ensure that the logo works for both light and dark mode.
